### ISSUE TITLE
Missing nautilus-megasync 3.6.6 in Archlinux repo

### ISSUE THREAD

**Issue by David-UPCT** [meganz/MEGAsync#211]
Hi. In https://mega.nz/linux/MEGAsync/Arch_Extra/x86_64/ there's no nautilus-megasync-x86_64.pkg.tar.xz file. I guess this file is missing, isn't it?

Cheers,

David.

**Comment by polmr** [comment]
You are right @David-UPCT , We will fix it asap, thanks for reporting that :+1:
